rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -613,6 +613,61 @@
             (BridgeMethodExecutorTyped) Bridge::registerFastBridgeBtcTransaction,
             activations -> activations.isActive(RSKIP176),
             false
+    ),
+    GET_BEST_BLOCK_HEADER(
+            CallTransaction.Function.fromSignature(
+                    "getBestBlockHeader",
+                    new String[0],
+                    new String[]{"bytes"}
+            ),
+            fixedCost(20000L),
+            (BridgeMethodExecutorTyped) Bridge::getBestBlockHeader,
+            activations -> true, // TODO: Review activation
```

Comment:
Just create a variable and add `activations.isActive(RSKIP_TMP_VAL)`

Reply:
Done, now I have the RSKIP number